code/game/objects/items/implants/implant_chem.dm

```diff
@@ -29,11 +29,8 @@
 	GLOB.tracked_chem_implants -= src
 	return ..()
 
-/obj/item/implant/chem/trigger(emote, mob/living/source)
-	if(emote == "deathgasp")
-		if(istype(source) && !(source.stat == DEAD))
-			return
-		activate(reagents.total_volume)
+/obj/item/implant/chem/on_mob_death(mob/living/L, gibbed) //yogs - chem implants dump chems when host dies
+	activate("action_button")
```

Comment:
```suggestion
	activate("death")
```

Reply:
I don't mind changing this, but I was wondering is this so its easier to read or is action_button used for something else in this implants case?

Reply:
Probably doesnt affect anything but its good practice to keep shit as its meant and not something random because you saw it like that somewhere else.

BTW,modularization is no longer enforced so you dont need to do the ``//yogs - chem implants dump chems when host dies``